fem/pbilinearform.hpp

```diff
@@ -295,15 +340,84 @@ class ParMixedBilinearForm : public MixedBilinearForm
    {
       trial_pfes = trial_fes;
       test_pfes  = test_fes;
+      keep_nbr_block = false;
    }
 
+   /** When set to true and the ParBilinearForm has interior face integrators,
+       the local SparseMatrix will include the rows (in addition to the columns)
+       corresponding to face-neighbor dofs. The default behavior is to disregard
+       those rows. Must be called before the first Assemble call. */
+   void KeepNbrBlock(bool knb = true) { keep_nbr_block = knb; }
+
+   /// Assemble the local matrix
+   void Assemble(int skip_zeros = 1);
+
    /// Returns the matrix assembled on the true dofs, i.e. P_test^t A P_trial.
-   HypreParMatrix *ParallelAssemble();
+   /** The returned matrix is owned by the form. */
+   HypreParMatrix *ParallelAssembleInternal();
```

Comment:
Similar to above: I don't think we want method called `*Internal` to be part of the public interface.

It is best to either move it to be protected or rename it.